README.md

```diff
@@ -58,7 +58,7 @@ An archive is available for people who don't want to install `atom` as root.
 This version enables you to install multiple Atom versions in parallel. It has been built on Ubuntu 64-bit,
 but should be compatible with other Linux distributions.
 
-1. Install dependencies (on Ubuntu): `sudo apt install git gconf2 gconf-service libgtk2.0-0 libudev1 libgcrypt20 libnotify4 libxtst6 libnss3 python3 gvfs-bin xdg-utils libcap2`
+1. Install dependencies (on Ubuntu): `sudo apt install git libgcrypt20, libgtk-3-0, libnotify4, libnss3, libglib2.0-bin, xdg-utils, libx11-xcb1, libxcb-dri3-0, libxss1, libxtst6, libxkbfile1, libcurl4`
    1. (If the `python3` package isn't available, or is too old (Python 3 should be >= 3.5), either `python2` or `python` (2.6 or 2.7) will work in its place.)
```

Comment:
I guess we can delete the note about `python3`/`python2`/`python` just below that.

The note is a bit out of place now that we don't tell people to go out of their way to install Python anymore.

I think the expectation is that anyone on a graphical Ubuntu desktop has Python 3 at this point. (Bionic is the oldest supported Ubuntu, and it has Python 3.6, which is new enough. https://packages.ubuntu.com/bionic/python3)

Alternatively, add a bit more info to the note: "You'll want to have a copy of Python for installing certain Atom packages. And a working `gcc`+`g++` or `clang` compiler installed as well."